frontend/__tests__/components/cluster-settings.spec.tsx

```diff
@@ -33,6 +34,12 @@ import {
 import { GlobalConfigPage } from '../../public/components/cluster-settings/global-config';
 import { Firehose, HorizontalNav, ResourceLink, Timestamp } from '../../public/components/utils';
 
+jest.spyOn(storeHandler, 'getStore').mockImplementation(() => ({
+  getState: () => ({
+    sdkCore: {},
+  }),
+}));
```

Comment:
`spyOn` can be moved inside describe block under `beforeEach/beforeEach` and should clear the mock as well as below and we use spy when we need to assert if what we are spying has been called or not. This applies to other test files as well.

```
 afterEach(() => {
    jest.resetAllMocks();
  });
 ```
 
 https://jestjs.io/docs/jest-object#jestspyonobject-methodname